lib/sqlalchemy/sql/selectable.py

```diff
@@ -175,9 +176,10 @@ def entity_namespace(self) -> _EntityNamespace: ...
     Union[
         "_ColumnExpressionArgument[Any]",
         "_FromClauseArgument",
+        Type[DeclarativeBase],
     ],
     # or sequence of single column elements
-    Sequence["_ColumnExpressionArgument[Any]"],
+     Sequence[Union["_ColumnExpressionArgument[Any]", Type[DeclarativeBase]]],
```

Comment:
this list needs `_FromClauseArgument` if the "of" can point to lists of tables, which will accommodate the ORM mapped classes as well